noxfile.py

```diff
@@ -39,6 +39,13 @@ def test_pythonpath(session):
     run_tests(session, env={'PYTHONPATH': 'src'})
 
 
+@nox.session(python='3.8')
+def docs(session):
```

Comment:
I haven't gone wild here -- there are actually a few different sphinx builders that I personally run in CI (e.g. linkcheck, which will ensure links are not broken, spelling, to ensure I haven't made any typos, etc.).

I kept it light for now but yeah there's more here if desirable.

Reply:
Ah, that is fine by me :grin: